bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
+    
+    public static void main(String[] args) {
+        ChildClass child = new ChildClass();
+        child.doSomething();
+        child.doSomethingElse();
+    }
+}
+
+class ParentClass {
+    public int someNumber = 10;
```

Comment:
Consider using a more descriptive name for the `someNumber` field, as it is used in both `ParentClass` and `ChildClass` with different meanings. [important]